sonar-project.properties

```diff
@@ -13,7 +13,7 @@ sonar.sourceEncoding=UTF-8
 # Path is relative to the sonar-project.properties file. Replace “\” by “/” on Windows.
 # This property is optional if sonar.modules is set.
 sonar.sources=src/app
-sonar.exclusions=test-config/**, cypress/**, dist/**, .nyc_output/**, .scannerwork/**, coverage/**, **/*.module.ts, **/*.routes.ts, **/*.state.ts, **/utils.ts
+sonar.exclusions=test-config/**, cypress/**, dist/**, .nyc_output/**, .scannerwork/**, coverage/**, **/*.module.ts, **/*.routes.ts, **/*.state.ts, **/utils.ts, **/*.utils.ts
```

Comment:
why this change?

Reply:
As per one of our discussions, I added the utils/test.utils to be ignored by sonar. Please let me know if you changed your mind.

Reply:
it doesn't need to be sonar ignored. the file is used sufficiently in unit tests hence should not affect coverage

Reply:
Sonar included the file. I can show you. We can ask Umer for his oppinion as well.